crates/xline/src/server/auth_server.rs

```diff
@@ -142,8 +142,8 @@ where
         &self,
         mut request: tonic::Request<AuthUserAddRequest>,
     ) -> Result<tonic::Response<AuthUserAddResponse>, tonic::Status> {
-        debug!("Receive AuthUserAddRequest {:?}", request);
         let user_add_req = request.get_mut();
+        debug!("Receive AuthUserAddRequest {}", user_add_req);
```

Comment:
It's in the `debug` tracing call. Why do you use display instead of debugging?

Reply:
Try to override this trait https://doc.rust-lang.org/std/fmt/trait.Debug.html

Reply:
The prost will automatically derive the debug trait for all the requests and responses it generates.  So far, we cannot switch off this behavior for the prost crate. FYI: https://github.com/tokio-rs/prost/issues/334. So I have to override the Display trait for it.